### PR TITLE
Badge and button

### PR DESCRIPTION
**Pull request by harshparab**
image
image
image
image
image
image
image
image

### CODE REVIEW

File: Components/button.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
+.button {
+    font-weight: 700;
+    background: var(--default-color);
+    color: var(--background-color);
+    width: max-content;
+    padding: 1rem 1rem ;
+    border-radius: 30px;
+    border: none;
```
Comment (Richa-rani02) [inline Components/button.css]: suggestion :Better to use rem

**Review by Richa-rani02 — CHANGES_REQUESTED** [review]
suggested some changes .rest looks good.

File: Components/button.css (hunk 1 above)
Comment (harshparab) [reply Components/button.css]: Okay sure, Thanks!

Comment (harshparab) [reply Components/button.css]: Done

File: Components/badge.html [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            <div class="left-nav">
+                <a href="../../index.html">Nuclei</a>
+            </div>
+            <ul class="right-nav">
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Github</a></li>
+            </ul>
+        </nav>
+    </div>    
+    <div class="doc-container">
```
Comment (adith-a-danthi) [inline Components/badge.html]: You can use `main` tag here

**Review by adith-a-danthi — APPROVED** [review]
Looks Good. Try to use semantic HTML tags like `main`, `header`, `aside` for page elements.

File: Components/badge.html (hunk 2 above)
Comment (harshparab) [reply Components/badge.html]: Used it

Comment (harshparab) [reply Components/badge.html]: Thanks

**Review by Richa-rani02 — APPROVED** [review]

**Pull request merged**